webpack.config.js

```diff
@@ -0,0 +1,81 @@
+const path = require('path');
```

Comment:
No need this file for intermediary module. please check `esn-frontend-inbox-calendar` for reference

Reply:
this file is removed